lab-jonathan/test/hex-test.js

```diff
@@ -0,0 +1,16 @@
+let hex = require('../lib/readHex.js');
+let assert = require('assert');
+
+
+describe('pulling first hex values from files', function(){
+  describe('readHex', function(){
+    it('should verify that the contents of allData are equal to what is expected', function(done){
+      hex.read(function(allData) {
+        assert.equal(hex.data[0], '5a6f6');
```

Comment:
Looking good on the tests. I'd rather that each assertion had it's own `it()` block, but this is functional as it is. Going forward, best to only test one thing at a time though.